frontend/src/main/components/Users/UsersTable.js

```diff
@@ -1,33 +1,102 @@
 import React from "react";
 import OurTable from "main/components/OurTable"
-
-const columns = [
-    {
-        Header: 'id',
-        accessor: 'id', // accessor is the "key" in the data
-    },
-    {
-        Header: 'First Name',
-        accessor: 'givenName',
-    },
-    {
-        Header: 'Last Name',
-        accessor: 'familyName',
-    },
-    {
-        Header: 'Email',
-        accessor: 'email',
-    },
-    {
-        Header: 'Admin',
-        id: 'admin',
-        accessor: (row, _rowIndex) => String(row.admin) // hack needed for boolean values to show up
-    },
-];
+import { Button } from "react-bootstrap";
+import { useBackendMutation } from "main/utils/useBackend";
+import { cellToAxiosParamsDelete, cellToAxiosParamsUnhide, onDeleteSuccess } from "main/utils/usersUtils"
```

Comment:
Can we rename `cellToAxiosParamsDelete` to 
`cellToAxiosParamsHide` to avoid confusion?

Same with `onDeleteSuccess`... shouldn't that be `onHideSuccess` ?

Reply:
Yes.